jenkins-scripts/dsl/_configs_/OSRFLinuxCompilation.groovy

```diff
@@ -49,9 +49,47 @@ class OSRFLinuxCompilation extends OSRFLinuxBase
     {
       publishers
       {
-         // compilers warnings
-         warnings(['GNU C Compiler 4 (gcc)'], null) {
-             thresholds(unstableTotal: [all: 0])
+          configure { project ->
+            project / publishers / 'io.jenkins.plugins.analysis.core.steps.IssuesRecorder' {
+              analysisTools {
+                'io.jenkins.plugins.analysis.warnings.Gcc4' {
+                  id()
+                  name()
+                  pattern()
+                  reportEncoding()
+                  skipSymbolicLinks(false)
+                }
+                'io.jenkins.plugins.analysis.warnings.Cmake' {
```

Comment:
I don't see the removed CMake warnings configuration in the removed lines, just the GCC warnings config. Am I missing something?

Reply:
You are right. I dropped a note in issue description

>     * The change will add cmake parser to our builds and probably trigger a good bunch of new warnings in some builds, I consider it as a desired improvement in QA and something maintainers probably want to fix.